OpenRA.Mods.Cnc/Activities/Teleport.cs

```diff
@@ -63,9 +63,14 @@ public override Activity Tick(Actor self)
 			Game.Sound.Play(SoundType.World, sound, self.CenterPosition);
 			Game.Sound.Play(SoundType.World, sound, self.World.Map.CenterOfCell(destination));
 
-			self.Trait<IPositionable>().SetPosition(self, destination);
-			self.Generation++;
+			var positionable = self.Trait<IPositionable>();
+positionable.SetPosition(self, destination);
```

Comment:
Indentation please.

Reply:
Also, spaces -> tabs please

Reply:
Is the only thing wrong with this pull request the way the code was styled?

Reply:
Can't really tell as I didn't test the code yet. However it was this style issue that left the PR without reviewing progress. So, fixing the style would be a good step towards resurrecting this. ^^